### PR TITLE
[bump] client: 2.0.0-internal.8.0.3 => 2.0.0-internal.8.0.4 (patch)

### PR DESCRIPTION
**Pull request by tyler-cai-microsoft**
Bumped client from 2.0.0-internal.8.0.3 to 2.0.0-internal.8.0.4.

### CODE REVIEW

**Review by alexvy86 — APPROVED** [review]
The build error I suspect is something that happens non-deterministically... I've seen the md file for mocha-test-setup sometimes change the order of things in a list or little details like that, that still result in the extraneous modified files check failing. We might need to do a full repo build locally for this branch, commit the change to that file, and try the PR build again.

**Comment by tyler-cai-microsoft** [comment]
I ran a local build, and I had no changes locally. This is weird

**Comment by alexvy86** [comment]
Ran a build in Codespaces and did get a change, I'll push it to the branch:

image

It's not the one I expected but I've also seen this one a few times.

File: packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md [hunk 1]
```diff
@@ -15,4 +15,4 @@ export const mochaHooks: {
 
 // (No @packageDocumentation comment for this package)
 
-```
+```
```
Comment (tyler-cai-microsoft) [inline packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md:18]: This is the troublesome commit

Comment (tyler-cai-microsoft) [reply packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md:18]: I'm going to recreate this bump from a codespaces

**Pull request closed without merging**